### PR TITLE
Small improvements to fairness of tests

### PR DESCRIPTION
**Pull request by lovell**
Hello, thanks for creating this benchmark. This PR contains a few small changes to ensure the tests are as close as possible to each other.

To start with, here are the times I see locally on an Intel i3-4170 CPU running Ubuntu 16.04 and using libvips 8.6.0 (master/HEAD) when running the original, unmodified tests:

```
ruby ruby_io_speed.rb
Took: 324ms
node node_io_speed.js
Took: 208.100ms
node node_io_speed_streams.js
Took: 373.145ms
```

The first thing I noticed was that the Node Streams version specifies the `max` option where the other tests do not. Correcting this slows it by ~10%:

```
ruby ruby_io_speed.rb
Took: 314ms
node node_io_speed.js
Took: 207.065ms
node node_io_speed_streams.js
Took: 412.869ms
```

Secondly I updated to the latest version of sharp, given the older version specified here had poorer stream performance and slightly poorer image quality. This brings the two Node tests much closer to each other.

```
ruby ruby_io_speed.rb
Took: 324ms
node node_io_speed.js
Took: 246.919ms
node node_io_speed_streams.js
Took: 262.504ms
```

Finally, SIMD is enabled for the ruby bindings by default but with the latest sharp is not, due mainly to a few reported crashes from liborc when under load. Enabling this produces:

```
ruby ruby_io_speed.rb
Took: 322ms
node node_io_speed.js
Took: 204.619ms
node node_io_speed_streams.js
Took: 224.569ms
```

Switching from the large "cakes" image to the smaller "bridge" image generates a lot more variance in the run times:

```
ruby ruby_io_speed.rb
Took: 90ms
node node_io_speed.js
Took: 112.317ms
node node_io_speed_streams.js
Took: 133.813ms
```

In summary, the winner here is libvips regardless of language used to access it :)

### CODE REVIEW

**Comment by shime** [comment]
Wow, thanks for the detailed report and explanations!

**Comment by lovell** [comment]
No worries, thanks for merging. I didn't update any of the README graphs as I imagine you'll want to do this yourself under the same conditions as previous test runs.
